SeniorInterview/StocksAnalyzed/src/AggregateRequestor.java

```diff
@@ -0,0 +1,59 @@
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.ArrayList;
+
+public class AggregateRequestor extends Requestor {
+
+    public String request(String ticker, String startDate, String endDate){
+        String grouppedURL = String.format("https://api.polygon.io/v2/aggs/ticker/%s/range/1/day/%s/%s?adjusted=true&sort=asc&limit=120&apiKey=%s",
+                ticker,
+                startDate,
+                endDate,
+                apiKey);
+
+        HttpRequest request = HttpRequest.newBuilder()
```

Comment:
You can create a new function on REquestor that receives the URI and method to create the httprequest, as this should probably be used by all resquestors